lib/utils.js

```diff
@@ -258,6 +258,19 @@ exports.autoStoreData = function (req, res, next) {
     req.session.data = {}
   }
 
+  try {
+    let defaultsFilePath = path.join(__dirname, '/../app/data/session-defaults.js')
```

Comment:
Requiring the file can come out of the middleware, it only needs to happen once right?